Cargo.toml

```diff
@@ -8,15 +8,15 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-lightning = { version = "0.0.106", features = ["max_level_trace"] }
-lightning-block-sync = { version = "0.0.106", features = [ "rpc-client" ] }
-lightning-invoice = { version = "0.14" }
-lightning-net-tokio = { version = "0.0.106" }
-lightning-persister = { version = "0.0.106" }
-lightning-background-processor = { version = "0.0.106" }
+lightning = { path= "../rust-lightning/lightning", version = "0.0.106", features = ["max_level_trace"] }
```

Comment:
Do we still need `version` when specifying path dependencies and should it be 0.0.107?

Reply:
Yeah, will switch back to only use the version and update to 107 once we cut a release.